definitions/pipelines/rnaseq.cwl

```diff
@@ -40,6 +40,9 @@ inputs:
         type: File
     gene_transcript_lookup_table:
         type: File
+    markdup_assume_sort_order:
+        type: string?
+        default: "coordinate"
```

Comment:
This doesn't need to be an input, since it's hardwired into the way this pipeline runs.